coralproject/talk
# [CORL-3172] optimized redis query for counts v2 endpoint

## What does this PR do?

- Replaces the multiple async mget promises with a singular redis mget with multiple key returns.
- Replaces the multiple async await with an optimized for loop

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

N/A

## How do I test this PR?

- Start up Coral
- Query the counts v2 endpoint with some storyID's
- See that it only makes one call to Redis to retrieve the storyID's

## Were any tests migrated to React Testing Library?

No

## How do we deploy this PR?

- Merge into https://github.com/coralproject/talk/pull/4635
